server_admin/topics/authentication/webauthn.adoc

```diff
@@ -167,27 +166,61 @@ A new user carries out the following operations :
 
 Existing user::
 
-When existing users try to log in, they are required to register their WebAuthn authenticator automatically :
+If `WebAuthn Authenticator` is set up as required as shown in the first example, then when existing users try to log in,
+they are required to register their WebAuthn authenticator automatically :
 
 - Open the login form.
 - Fill in items, click `Save` and  click `Login`.
 - When the users log in, they are required to register their WebAuthn authenticator.
 - After successful registration, the user's browser asks the user to enter the text as their just registered WebAuthn authenticator's label.
 
-===== View Registered WebAuthn Authenticator
+==== Passwordless WebAuthn
```

Comment:
Thanks. I've used "Passwordless WebAuthn together with Two-Factor" .This is due the main point of this example is to introduce "Passwordless WebAuthn" as the authentication method and the two-factor is here just because it was configured previously. It doesn't matter which two-factor mechanism exactly is used. The fact that WebAuthn is used as two-factor is just because it was easier to extend this section from previous example.